applications/NXem.nxdl.xml

```diff
@@ -0,0 +1,1693 @@
+                        <item value="front_bottom_left"/>
+                        <item value="back_top_left"/>
+                        <item value="back_top_right"/>
+                        <item value="back_bottom_right"/>
+                        <item value="back_bottom_left"/>
+                    </enumeration>
+                </field>
+                <field name="x_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing x-axis base vector of the
+                        sample_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+                <field name="y_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing y-axis base vector of the
+                        sample_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+                <field name="z_direction" type="NX_CHAR" recommended="true">
+                    <doc>
+                        Direction of the positively pointing z-axis base vector of the
+                        sample_reference_frame.
+                    </doc>
+                    <enumeration>
+                        <item value="north"/>
+                        <item value="east"/>
+                        <item value="south"/>
+                        <item value="west"/>
+                        <item value="in"/>
+                        <item value="out"/>
+                    </enumeration>
+                </field>
+            </group>
+            <group name="DETECTOR_REFERENCE_FRAME" type="NXcoordinate_system" nameType="any" minOccurs="0" maxOccurs="unbounded">
+                <doc>
+                    Instances should use detector_reference_frame as a name prefix.
+                </doc>
+                <field name="depends_on" type="NX_CHAR" optional="true">
+                    <doc>
+                        Reference to the specifically named :ref:`NXdetector` instance for
+                        which these conventions apply (e.g. /entry1/instrument/detector1).
+                        
+                        Instances should use detector_reference_frame as a name prefix.
```

Comment:
So should this be detector_reference_frame_DETECTOR_REFERENCE_FRAME with nameType="partial"?

Reply:
Yes, like for all other cases of this pattern, I will resolve all matching comments on these as these have been reworked